src/components/Footer/Footer.css

```diff
@@ -0,0 +1,67 @@
+.footer {
+  display: flex;
```

Comment:
You can use tailwind for this property, see: https://tailwindcss.com/docs/display